packages/wrangler/src/utils/shell-quote.ts

```diff
@@ -3,6 +3,15 @@ import shellquote from "shell-quote";
 export const quote = shellquote.quote;
 
 export function parse(cmd: string, env?: Record<string, string>): string[] {
+	// This is a workaround for a bug in shell-quote on Windows
+	// It was fixed and then reverted in https://github.com/ljharb/shell-quote/commit/144e1c20cd57549a414c827fb3032e60b7b8721c#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L16
```

Comment:
as before, if you want we can shorten the url I think 🙂 
```suggestion
	// It was fixed and then reverted in https://github.com/ljharb/shell-quote/commit/144e1c2#diff-e727e4bL16
```